robottelo/ui/base.py

```diff
@@ -539,11 +540,14 @@ def click(self, locator, wait_for_ajax=True,
         :raise: UINoSuchElementError if the element could not be found.
 
         """
-        element = self.wait_until_element(locator, timeout=waiter_timeout)
+        if type(locator).__name__ == 'tuple':
```

Comment:
Yep, just realized that and wanted to update my comment, but you're faster :smile: 
Well, we could try something like this:
```python
if type(locator) == WebElement or locator is None:
```
but that looks not that good. On the other hand, this way we'd directly specify all 3 possible cases so we won't forget anything for later changes, for example.
Anyway, you can skip this suggestion if you want, as it appeared not that cool :)